src/js/utils/ServiceSpecUtil.js

```diff
@@ -0,0 +1,33 @@
+import PodSpec from '../structs/PodSpec';
+import Util from './Util';
+
+var ServiceSpecUtil = {
+
+  /**
+   * Creates a new PodSpec with the scaling specifications
+   * changed to fixed, and with the number of instances defined
+   * from the second argument.
+   *
+   * @param {PodSpec} spec - The spec to change
+   * @param {number} instancesCount - The new number of instances
+   * @returns {PodSpec} - Returns a new PodSpec instance
+   */
+  setFixedScaling(spec, instancesCount) {
```

Comment:
Would love if we could make this a general scale service method that know how to deal with pods and apps? That would allow us to remove all the type checks form the views.
What do you think?